CMakeLists.txt

```diff
@@ -17,18 +17,25 @@ else()
   target_compile_options(project_warnings INTERFACE -Wall -Wextra -Wpedantic)
 endif()
 
-add_executable(intro main.cpp)
+#qt
+find_package( Qt5Widgets REQUIRED )
+set( CMAKE_AUTOMOC ON )
+add_executable( helloQt qt/qtMain.cpp qt/HelloQt.cpp )
+target_link_libraries( helloQt Qt5::Widgets )
+target_compile_features( helloQt PUBLIC cxx_nullptr )
+
+add_executable(intro main.cpp )
 target_compile_features(intro PRIVATE cxx_lambda_init_captures)
 target_link_libraries(intro PRIVATE project_warnings)
 
-target_link_libraries(intro --coverage)
+#target_link_libraries(intro --coverage)
 
 enable_testing()
 
 add_executable(tester tester.cpp)
 target_link_libraries(tester PRIVATE project_warnings)
 
-target_link_libraries(tester --coverage)
+#target_link_libraries(tester --coverage)
 add_test(Tester tester)
 
```

Comment:
I recommend putting a CMakeLists.txt into your /qt folder as there can be a lot of options for different platforms. Please feel free to copy the CMakeLists.txt from my minimal project at https://github.com/alexallmont/MinimalQt/blob/master/CMakeLists.txt. This prevent dialogs from popping up in OSX and ensures all the Qt dlls are deployed against the generated exe.